cli/src/command_error.rs

```diff
@@ -375,7 +375,20 @@ impl From<ConflictResolveError> for CommandError {
 
 impl From<MergeToolConfigError> for CommandError {
     fn from(err: MergeToolConfigError) -> Self {
-        user_error_with_message("Failed to load tool configuration", err)
+        match err {
```

Comment:
nit: `&err` (and remove `ref`)?

Reply:
Seems to work, thanks! This Rust detail still confuses me.